Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        //Return Str Function
+        public static string printStr(string message, string variable)
+        {
+            Console.WriteLine(message);
+            variable = Console.ReadLine();
+            if (string.IsNullOrEmpty(variable)) 
+            {
+                Console.WriteLine("Name cannot be empty");
+                return "-";                  
+            }
+            return variable;
+        }
+
+        //Return Int Function
+        public static int printInt(string message, int variable)
+        {
+            Console.WriteLine(message);
+            var input = Console.ReadLine();
+            var isNumber = int.TryParse(input, out var age1);
+            if (!isNumber)
+            {
+                Console.WriteLine(input + " is not a valid number");
+                return -1;
+            }
+            return variable = age1;
+        }
+
+        //Return Float Function
```

Comment:
Such comments don't add any more information than the code already says. Can be removed.
It's okay to have such comments initially, as placeholders, but they should be pushed for publication